script.js

```diff
@@ -0,0 +1,35 @@
+// Mouse Gun №8
+// const body = document.querySelector('body');
+// body.addEventListener('click', function (e) {
+//     const elem = document.elementFromPoint(e.clientX, e.clientY);
+//     elem.style.visibility = "hidden";
+//     console.log(`X = ${e.clientX} Y = ${e.clientY}`);
+// });
+
+
+// Mouse Gun №9
+const body = document.querySelector('body');
+let elem ;
+
+const listener = function (event) {
```

Comment:
do not need to assign function to const use arrow function or named function instead